apps/nextra/pages/en/build/smart-contracts/book/structs-and-resources.mdx

```diff
@@ -421,9 +424,96 @@ module 0x2::m {
 }
 ```
 
+## Positional Structs
+
+_Since language version 2.0_
+
+A struct can be declared to have _positional fields_, fields which are not named
+but numbered. Positional structs behave similar to regular structs,
+except providing a different syntax which is more adequate for use cases with 
+only a few fields.
+
+Fields of positional structs are assigned in the order they appear. In the below
+example, field `0` is of type `u64` and field `1` of type `u8`:
+
+```move
+module 0x2::m {
+  struct Pair(u64, u8);
+}
+```
+
+Abilities for positional structs are declared _after_ and not before the field list,
+which is expected to be short:
+
+```move
+module 0x2::m {
+  struct Pair(u64, u8) has copy, drop;
+}
+```
+
+For pure type tags, often used for phantom types in Move code, the list of arguments
+can be also completely omitted:
+
+```move
+module 0x2::m {
+  struct TypeTag has copy, drop;
+}
+```
+
+Values of positional structs are created and deconstructed similar as function calls,
+using `PositionalStructs(arguments)`:
+
+```move
+module 0x2::m {
+  fun work() {
+    let value = Pair(1, true);
+    let Pair(number, boolean) = value;
+    assert!(number == 1 && boolean == true);
+  }
+}
+```
+
+Fields of positional structs can be accessed using the position as a field selector, as in
+`positional_struct.0`.
+
+## Partial Patterns
+
+_Since language version 2.0_
+
+Patterns can use the `..` notation to match any remaining, non-listed fields. Here are 
+some examples:
+
+```move
+module 0x2::m {
+  struct Foo{ x: u8, y: u16, z: u32 }
+  struct Bar(u8, u16, u32);
+  
+  fun foo_get_x(self: &Foo): u8 { 
+    let Foo{x, ..} = self;
```

Comment:
```suggestion
    let Foo{y, ..} = self; y
```

Correspondingly change the return type. Just to show that it not necessarily every field from the beginning in the case of named fields.